lib/middleware/backbone_request_id.js

```diff
@@ -0,0 +1,11 @@
+import Backbone from 'backbone'
+import { wrap } from 'underscore'
+
+module.exports = (req, res, next) => {
+  Backbone.sync = wrap(Backbone.sync, (sync, method, model, options = {}) => {
```

Comment:
Unfortunately, model-level `sync` wouldn't be much different as models are also global libraries. The tricky part is we need to send request-level runtime information to compile-time global libraries. In a synchronous environment like OOTB Rails _maybe_ you could hack through this limitation because you stop the entire world for every request and could count on a global GUID generated before and after a request finishes to be the same—but b/c Node is async we have to tie these IDs to the incoming request (hence middleware) and then to the individual outgoing requests (via superagent in Backbone + Metaphysics).

You could get clever about tying the `req` info to make it more search-replace friendly by providing a helper like `model = new req.Artwork ...`, but that's just a different API to the same solution.

I suggest we punt on passing through the Backbone requests on this PR and just start with the metaphysics code.